odoo/addons/base/models/res_partner.py

```diff
@@ -392,6 +393,17 @@ def _compute_company_registry(self):
         for company in self:
             company.company_registry = company.company_registry
 
+    @api.depends('company_id', 'country_id', 'country_id.vat_label')
```

Comment:
I think `'country_id',` is redundant with `'country_id.vat_label'`.

Except that it is ok for me.

Reply:
Yes, you are right, country and vat labels are created by the system. generally, it's not changed.